pkgs/development/python-modules/pystray/default.nix

```diff
@@ -12,8 +22,8 @@ buildPythonPackage rec {
     sha256 = "sha256-8B178MSe4ujlnGBmQhIu+BoAh1doP9V5cL0ermLQTvs=";
   };
 
-  nativeBuildInputs = [ sphinx ];
-  propagatedBuildInputs = [ pillow xlib six ];
+  nativeBuildInputs = [ gobject-introspection sphinx ];
+  propagatedBuildInputs = [ pillow xlib six pygobject3 gtk3 libayatana-appindicator ];
```

Comment:
Are the scripts in $out/bin now double wrapped?

Reply:
No, I changed the two applications relying on pystray (which is just a package) to not wrapGapps and added the paths like other python applications does.